core/iOrmset.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/*
+ * @copyright   Copyright (C) 2010-2022 Combodo SARL
+ * @license     http://opensource.org/licenses/AGPL-3.0
+ */
+
+/*
+ * this interface add a function witch allow import on object (like ormIPv6)
+ * @since 3.1.0
+ */
+
+interface iOrmset
```

Comment:
I don't think this is actually a set of ORM objects? If it is, shouldn't the naming more be like this?
```suggestion
interface iOrmSet
```

Reply:
Hi @accognet,  you resolved, but I didn't see an answer to my question?